scripts/update-monitoring-mixin-dashboards.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
How does this script compare with [scripts/sync-mixins.sh](https://github.com/giantswarm/dashboards/blob/master/scripts/sync-mixins.sh)?
Could you add some documentation explaining when we should use one or the other?

Reply:
Please check the new docs

Reply:
I'm not happy with the way things are moving regarding mixins.
We've talked about it, but I feel like I have to write it down.

## Some background about mixins

Mixins are templates, that we can build to our needs.
So: mixins + config => custom-fit dashboards

But, like with helm charts, not all mixins are the same quality, not all follow the same coding/config conventions.
And, unlike helm charts, it's not that easy to generate them.

So, you can find pre-generated mixins at https://monitoring.mixins.dev/ that may fit, or not.
Even if they don't perfectly fit, it's often easier to do like with [grafana dashboards](https://grafana.com/grafana/dashboards/): import, then manually edit.

## Mixins at giantswarm

All I can say is it's very inconsistent :( 
- Some are generated from mixins (is that what happens here? https://github.com/giantswarm/giantswarm-kubernetes-mixin)
- Some are manually imported and tuned
- and for some others that look to need no tuning, we are building a job to take them as-is from https://monitoring.mixins.dev/

I don't want to spend too much time figuring out what is happening where, should be part of this issue: https://github.com/giantswarm/giantswarm/issues/22481

## Conclusion

I understand that there's no silver bullet, and each dashboard has different challenges in their path from upstream mixins to GS dashboards.
It's fine with me if we have different paths for now, as long as it's documented :angel: 
It's great with me if we have some ways to automate updates, like here for alertmanager dashboard.

But don't assume it will work like that for all mixins dashboards.
Each script / automation has to show clearly what dashboards it's been designed for, IMO.

Reply:
Thanks 😊 
You're right!  we don't have a centralized repo that we can use to track all mixins and updates


Can you add this topic for discussion in the refinement!